modules/core/petra/petra-sql-dsl-spi/src/test/java/com/liferay/petra/sql/dsl/spi/SQLDSLTest.java

```diff
@@ -414,6 +414,24 @@ public void testDerivedTable() {
 			defaultASTNodeListener.getTableNames());
 	}
 
+	@Test
+	public void testDSLFunctionParentheses() {
+		DSLFunction<Long> rightDSLFunction =
+			(DSLFunction<Long>)DSLFunctionFactoryUtil.add(
+				MainExampleTable.INSTANCE.mainExampleIdColumn, 2L);
+
+		rightDSLFunction = rightDSLFunction.withParentheses();
+
+		DSLFunction<Long> leftDSLFunction =
+			(DSLFunction<Long>)DSLFunctionFactoryUtil.multiply(
+				MainExampleTable.INSTANCE.mainExampleIdColumn,
+				rightDSLFunction.withParentheses());
```

Comment:
Sorry @carolmariaabb , this code should not be on this branch, I have another branch that I'm going to send just this commit to Core Infra review (because I'm changing Petra framework). After their review, I will send another PR to Objects just giving support for parentheses on `DDMExpressionDSLExpressionVisitor`.
Regarding your comment, I needed to repeat the invocation of that method to cover all lines of the class DSLFunction in unit test. If you check SQLDSLTest test, it uses CodeCoverageAssertor that enforces that for many classes, including DSLFunction.
The if is not covered without these 2 invocations, I don't know if you have another solution for that in test.
```
	public DSLFunction<T> withParentheses() {
		if (_wrapParentheses) {
			return this;
		}

		return new DSLFunction<>(_dslFunctionType, _expressions, true);
	}
```
I will drop the last 2 commits, thanks!